static/styles/message.css

```diff
@@ -0,0 +1,120 @@
+$first_column_width: 46px;
+
+.message_row {
+    &.include-sender {
+        .messagebox .messagebox-content {
+            grid-template-rows: 25px repeat(3, auto);
+            padding-top: 4px;
+
+            .message_content {
+                padding: 0;
+                grid-row-start: 2;
+            }
+
+            .message_time {
+                align-self: center;
+                position: unset;
+                margin-top: 1px;
+            }
+
+            .message_edit_notice {
+                position: unset;
+                align-self: center;
+            }
+
+            .message_controls {
+                align-self: center;
+                position: unset;
+            }
+
+            .message_sender {
+                overflow: hidden;
+                text-overflow: ellipsis;
+                grid-column: 1 / 3;
+                grid-row: 1 / 2;
+
+                > span {
+                    display: flex;
+                    margin-top: 6px;
+                }
+
+                .inline_profile_picture {
+                    flex-shrink: 0;
+                    /* Let user profile take extra height without
+                       having any affect on height of the container. */
+                    position: absolute;
+                }
+
+                .sender_name {
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    line-height: 1;
+                    /* Add padding to align user name with the content */
+                    padding-left: $first_column_width;
+                    outline: none;
+                }
+            }
+        }
+    }
+
+    .messagebox .messagebox-content {
+        display: grid;
+        align-items: start;
+        padding-left: 10px;
+        padding-right: 2px; /* For message focused border */
+        grid-template-rows: repeat(4, auto);
+        grid-template-columns: $first_column_width auto 60px 64px;
```

Comment:
I had to use these `60` and `64` magic numbers here to fit `message_controls` and time.

Reply:
Okay.

Reply:
Is there a way we can make them a `calc()` function of the sizes used for the message controls regions? That would be a lot more maintainable than comments.

Reply:
124px is where we want to be at for `message_time` + `message_controls` looking at our main branch. So, instead of using these numbers, I set these to `max-content` which works well for mobile widths too. I added padding around message time and message controls to get that desired 124px value on wide widths. Since these elements don't change width with a change in language, we should be fine. Although, time can occupy few pixels more or less width based on what time the message was sent since "10:00" occupies more width than "1:11".